app/views/bus_stops/by_sequence.haml

```diff
@@ -0,0 +1,13 @@
+.page-title Route #{@route.number} Bus Stops
+
+.container
```

Comment:
Confused by why we need a container at this level. Isn't there one in the layout?